src/components/_materialUi/TextField/index.stories.template.tsx

```diff
@@ -24,16 +25,22 @@ export const Template: Story<TextFieldProps> = ({
       onChange={(e: any): void => {
         setTextValue(e.target.value);
       }}
+      placeholder="Placeholder"
+      type="text"
       value={textValue}
       {...props}
     />
   );
 };
 
+const values = ['One', 'Two', 'Three', 'Four', 'Five'];
+
 export const SelectTemplate: Story<TextFieldProps> = (
   props: TextFieldProps,
 ) => {
-  const [value, setValue] = React.useState(1);
+  const [value, setValue] = React.useState<string | string[]>(
+    get(props, 'SelectProps.multiple') ? ['One'] : 'One',
```

Comment:
Any reason to use `get` here over the more type safe:

```suggestion
    props.SelectProps?.multiple ? ['One'] : 'One',
```

Reply:
No particular reason, just a habit! Will change